source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst

```diff
@@ -0,0 +1,208 @@
+===============================
+``mc admin prometheus metrics``
+===============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin prometheus metrics
+
+Description
+-----------
+
+.. start-mc-admin-prometheus-metrics-desc
+
+The :mc:`mc admin prometheus metrics` command prints Prometheus metrics for a cluster.
+
+.. end-mc-admin-prometheus-metrics-desc
+
+For more complete documentation on using MinIO with Prometheus, see :ref:`How to monitor MinIO server with Prometheus <minio-metrics-collect-using-prometheus>`
+
+.. admonition:: Use ``mc admin`` on MinIO Deployments Only
+   :class: note
+
+   .. include:: /includes/facts-mc-admin.rst
+      :start-after: start-minio-only
+      :end-before: end-minio-only
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command prints cluster metrics from the deployment at :term:`alias` ``myminio``:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc admin prometheus metrics myminio cluster
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] admin prometheus metrics  \
+                                           ALIAS    \
+                                           [TYPE]
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: ALIAS
+   :required:
+
+   The :mc:`alias <mc alias>` of a configured MinIO deployment for which the command prints metrics.
+
+.. mc-cmd:: TYPE
+   :optional:
+
+   The type of metrics to print.
+
+      .. versionchanged:: RELEASE.2023-10-07T15-07-38Z
+
+         ``resource`` metrics added
+
+      Valid values are:
+
+      - ``bucket``
+      - ``cluster``
+      - ``node``
+      - ``resource``
+
+      If not specified, the command returns cluster metrics.
+      Cluster metrics also include node metrics.
+      The output includes additional information about each metric, such as if its value is a ``counter`` or ``gauge``.
```

Comment:
a prom-knowledgeable person would already know this, but it helps others understand what to expect.